client/client.go

```diff
@@ -18,14 +18,22 @@ import (
 	"github.com/valyala/fasthttp"
 )
 
+type Logger interface {
```

Comment:
I think logger interface can be similar to https://github.com/go-resty/resty/blob/master/util.go#L30. It would give more customization for logging levels

Reply:
I'll take a look.